### DISCUSSION TITLE
Why do I need an approved projectId?

### DISCUSSION THREAD

**Discussion by davidescicolone** [WalletConnect/walletconnect-monorepo#2332] [Developer Support] [answered]
Dear community,
in our web app we are enabling the "connect with WalletConnect" feature. 
We have created a WallectConnect project Id and is correctly working in our dev environment.
Silly question: why we should submit the project Id for approval? Are there some advantages? Will the non approved project id be deactivated after a while?
Sorry we are quite new in web3...
Thank you for your support!
Davide

**Comment by finessevanes** [comment, marked as answer]
Hey there - happy to answer your questions.

>why we should submit the project Id for approval?

If you want to be listed in our [explorer](explorer.wallletconnect.com), then you want to submit your project.

>Are there some advantages?

If you're building a wallet, it's a bit more critical for you to submit your project so that users can find your wallet in our modal.  If you're building a dapp, there isn't much upside besides being listed on the explorer page.

>Will the non-approved project id be deactivated after a while?

Nope! As of today, there is no plan to deactivate projects that haven't formally been submitted and approved.
